NitroxPatcher/Patches/Dynamic/EscapePod_RespawnPlayer_Patch.cs

```diff
@@ -0,0 +1,20 @@
+using System.Reflection;
+using HarmonyLib;
+using NitroxClient.GameLogic;
+using NitroxModel.DataStructures;
+using NitroxModel.DataStructures.Util;
+using NitroxModel.Helper;
+
+namespace NitroxPatcher.Patches.Dynamic;
+
+public sealed partial class EscapePod_RespawnPlayer_Patch : NitroxPatch, IDynamicPatch
+{
+    private static readonly MethodInfo TARGET_METHOD = Reflect.Method((EscapePod t) => t.RespawnPlayer());
+
+    public static void Postfix(EscapePod __instance)
+    {
+        // LifePod.RespawnPlayer() runs both for player respawn (Player.MovePlayerToRespawnPoint()) and for warpme command
```

Comment:
```suggestion
        // EscapePod.RespawnPlayer() runs both for player respawn (Player.MovePlayerToRespawnPoint()) and for warpme command
```

Reply:
Do we not want to get this resolved? Code wise other than this I am content.